config/user.m4

```diff
@@ -17,15 +17,18 @@ AC_DEFUN([ZFS_AC_CONFIG_USER], [
 		ZFS_AC_CONFIG_USER_LIBUDEV
 		ZFS_AC_CONFIG_USER_LIBUUID
 		ZFS_AC_CONFIG_USER_LIBBLKID
+		ZFS_AC_CONFIG_USER_PAM
```

Comment:
and should this not have been moved into the linux conditional?

Reply:
```
        AM_COND_IF([BUILD_LINUX], [
                ZFS_AC_CONFIG_USER_UDEV
                ZFS_AC_CONFIG_USER_SYSTEMD
                ZFS_AC_CONFIG_USER_LIBUDEV
                ZFS_AC_CONFIG_USER_LIBUUID
                ZFS_AC_CONFIG_USER_LIBBLKID
                ZFS_AC_CONFIG_USER_PAM
        ])
```
 ?